### PR TITLE
init Diamond Assurance performance test

### PR DESCRIPTION
**Pull request by jerrytt**
[Initialize Diamond Assurance performance test]

- Related ticket: NA
- Needles: NA
- Verification run: http://openqa.da.suse.cz/tests/135

### CODE REVIEW

**Comment by jerrytt** [comment]
@alice-suse would you help to review this ? Thanks

File: tests/kernel_performance/da_perf.pm [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright 2012-2018 SUSE LLC
```
Comment (alice-suse) [inline tests/kernel_performance/da_perf.pm:3]: I think it should be 2022?

File: tests/kernel_performance/da_perf.pm [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# Maintainer: jtang@suse.com
+package da_perf;
+use strict;
+use warnings;
+use testapi;
+use base "consoletest";
+sub run {
+    my $self = shift;
+    my $wait_switch;
+    #read VAR
```
Comment (alice-suse) [inline tests/kernel_performance/da_perf.pm:16]: Suggest to use comment format "# Comment"

File: tests/kernel_performance/da_perf.pm [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    my $product_name = get_var('VERSION');
+    my $build = get_var('BUILD');
+    my $hosts_ip = get_var('DA_SUT_IP');
+    my $hosts_name = get_var('DA_HOSTNAME');
+    my $hypervisor = get_var('DA_XEN_HYPERVISOR');
+    #build cmd
```
Comment (alice-suse) [inline tests/kernel_performance/da_perf.pm:22]: same here

File: schedule/da-perf/deploy_sleperf.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+name: deploy_sleperf
+description: >
+  Maintainer: jtang@suse.com
+  deploy and run sleperf for Diamond Assurance
```
Comment (alice-suse) [inline schedule/da-perf/deploy_sleperf.yaml:5]: Suggest to capitalize to Deploy.

**Comment by jerrytt** [comment]
Thanks @alice-suse  updated

**Comment by alice-suse** [comment]
LGTM

**Pull request merged**